doc/widgets/explain-pred.md

```diff
@@ -0,0 +1,36 @@
+Explain Prediction
+==================
+
+Explains which features contribute the most to the prediction for a single instance based on the model and how they contribute.
+
+**Inputs**
+
+- Model: a model whose predictions are explained by the widget
+- Background data: data needed to compute explanations
+- Data: Single data instance whose prediction is explained by the widget
+
+**Outputs**
+
+- Scores: The SHAP value of each features value. Features that contribute more to prediction have higher score deviation from the 0.
+
+**Explain Prediction** widget explains classification or regression model's prediction for the provided data instance. The widget shows what features affect the prediction of selected class the most and how they contribute (towards or against the prediction). The explanation is computed with removing features, replacing them with different options from the background data, and observing the change in the prediction.
+![](images/Explain-Prediction.png)
+
+1. Select the target class -- the plot will show explanations for this class.
+2. Zoom in/out the plot.
+3. Observe the prediction probability for a class and base vale -- an average probability in the dataset.
+4. Plot which shows features that affect the prediction the most (features with longer tape length) and how they affect it. Red features increase the probability for a selected class while blue features decrease the probability. On the right from the tape, you can see the feature name and its value\* for the selected instance. The length of the tape segment (and number on the tape) represent the SHAP value for feature contribution -- it is how much feature the feature affects the probability for the selected class. 
+6. Get help, save the plot, make the report, or observe the size of input and output data.
+
+\* Some models (including logistic regression) extend the categorical feature to more features with the technique named [one-hot encoding](https://en.wikipedia.org/wiki/One-hot). It means each value in the feature gets a new column which has value 0 (the instance have no this feature value) or 1 (the instance have this feature value) for each instance. In those cases categorical features will be labeled with the format `feature-name=feature-value = 0/1` -- e.g. `chest pain=asymptomatic = 1`. It means that the feature chest pain has value asymptomatic. Model, in this case, made more columns for feature chest pain, one of them was asymptomatic, and it was the case for the selected data instance.
```

Comment:
the instance have no this feature value --> the instance does not have this feature value

Reply:
the instance have this feature value --> the instance has this feature value